tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/upgrade-abort-test.go

```diff
@@ -36,120 +32,63 @@ var _ = Describe(
 			time.Sleep(10 * time.Second)
 		})
 
-		It("Abort at IBU upgrade stage", reportxml.ID("69055"), func() {
-			By("Creating, enabling ibu pre-prep CGU and waiting for CGU status to report completed", func() {
-				prePrepCguBuilder := cgu.NewCguBuilder(cnfinittools.TargetHubAPIClient,
-					tsparams.PrePrepCguName, tsparams.IbuCguNamespace, 1).
-					WithCluster(tsparams.TargetSnoClusterName).
-					WithManagedPolicy(tsparams.PrePrepPolicyName).
-					WithCanary(tsparams.TargetSnoClusterName)
-				prePrepCguBuilder.Definition.Spec.Enable = ptr.To(true)
+		It("Aborts an upgrade at IBU upgrade stage", reportxml.ID("69055"), func() {
+			By("Creating an upgrade IBGU", func() {
 
-				prePrepCguBuilder, err := prePrepCguBuilder.Create()
-				Expect(err).ToNot(HaveOccurred(), "Failed to create pre-prep CGU.")
+				newIbguBuilder = ibgu.NewIbguBuilder(cnfinittools.TargetHubAPIClient,
+					tsparams.IbguName, tsparams.IbguNamespace).
+					WithClusterLabelSelectors(tsparams.ClusterLabelSelector).
+					WithOadpContent("oadp-cm", "ztp-group").
+					WithSeedImageRef("registry.kni-qe-18.lab.eng.tlv2.redhat.com:5000/ibu/seed:4.17.0-rc.1", "4.17.0-rc.1").
+					WithPlan([]string{"Prep"}, 20, 20).
+					WithPlan([]string{"Upgrade"}, 20, 20).
+					WithPlan([]string{"FinalizeUpgrade"}, 20, 20)
 
-				_, err = prePrepCguBuilder.WaitUntilComplete(10 * time.Minute)
-				Expect(err).ToNot(HaveOccurred(), "Pre-prep CGU did not complete in time.")
-			})
+				newIbguBuilder, err = newIbguBuilder.Create()
+				Expect(err).ToNot(HaveOccurred(), "Failed to create IBGU")
 
-			By("Creating, enabling ibu prep CGU and waiting for CGU status to report completed", func() {
-				prepCguBuilder := cgu.NewCguBuilder(cnfinittools.TargetHubAPIClient,
-					tsparams.PrepCguName, tsparams.IbuCguNamespace, 1).
-					WithCluster(tsparams.TargetSnoClusterName).
-					WithManagedPolicy(tsparams.PrepPolicyName).
-					WithCanary(tsparams.TargetSnoClusterName)
-				prepCguBuilder.Definition.Spec.Enable = ptr.To(true)
+			})
 
-				prepCguBuilder, err := prepCguBuilder.Create()
-				Expect(err).ToNot(HaveOccurred(), "Failed to create prep CGU.")
+			By("Aborting the upgrade phase once prep phase has finished", func() {
 
 				_, err = ibu.WaitUntilStageComplete("Prep")
 				Expect(err).NotTo(HaveOccurred(), "error waiting for prep stage to complete")
 
-				_, err = prepCguBuilder.WaitUntilComplete(25 * time.Minute)
-				Expect(err).ToNot(HaveOccurred(), "Prep CGU did not complete in time.")
-			})
-
-			By("Creating, and enabling ibu upgrade CGU", func() {
-				upgradeCguBuilder := cgu.NewCguBuilder(cnfinittools.TargetHubAPIClient,
-					tsparams.UpgradeCguName, tsparams.IbuCguNamespace, 1).
-					WithCluster(tsparams.TargetSnoClusterName).
-					WithManagedPolicy(tsparams.UpgradePolicyName).
-					WithCanary(tsparams.TargetSnoClusterName)
-				upgradeCguBuilder.Definition.Spec.Enable = ptr.To(true)
+				newIbguBuilder := ibgu.NewIbguBuilder(cnfinittools.TargetHubAPIClient,
```

Comment:
Before  creating the abort IBGU maybe we should delete the upgrade IBGU to ensure they're not colliding and avoid  potential race conditions?

Reply:
@josclark42 tnx, if it's expected then looks good.